packages/sdks/snippets/react/src/components/CustomColumns.tsx

```diff
@@ -21,15 +28,19 @@ const CustomColumns = (props: CustomColumnsProps) => {
       }}
     >
       <Blocks
-        blocks={props.columns[0]?.blocks}
-        path={`component.options.columns.0.blocks`}
+        blocks={props.column1.blocks}
+        path={`component.options.column1.blocks`}
         parent={props.builderBlock.id}
+        context={props.builderContext}
+        registeredComponents={props.builderComponents}
       />
 
       <Blocks
-        blocks={props.columns[1]?.blocks}
-        path={`component.options.columns.1.blocks`}
+        blocks={props.column2.blocks}
+        path={`component.options.column2.blocks`}
         parent={props.builderBlock.id}
+        context={props.builderContext}
+        registeredComponents={props.builderComponents}
```

Comment:
do we need this for react gen2? i thought we only needed it in angular gen2 due to specific limitations in angular sdk

Reply:
I see. I have removed it already and pushed the code.